llpc/context/llpcPipelineContext.cpp

```diff
@@ -999,6 +999,12 @@ void PipelineContext::setColorExportState(Pipeline *pipeline, Util::MetroHash64
     }
   }
 
+  if (state.alphaToCoverageEnable && formats.empty()) {
+    // NOTE: We must export alpha channel for alpha to coverage, if there is no color export,
+    // we force a dummy color export.
+    formats.push_back({BufDataFormat32, BufNumFormatUint});
```

Comment:
Why is this using uint and not one of the float-like data formats? Alpha-to-coverage doesn't work with an integer render target, does it?

Reply:
Using `uint` or `float` seems okay.  We just need to export a format with alpha channel for Alpha-to-coverage.